app/bundles/LeadBundle/Resources/views/Tokens/tokens_help_bundle.html.twig

```diff
@@ -0,0 +1,303 @@
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.id'|trans }}</td>
+                    <td><code>{contactfield=id}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.email'|trans }}</td>
+                    <td><code>{contactfield=email}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.facebook'|trans }}</td>
+                    <td><code>{contactfield=facebook}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.fax'|trans }}</td>
+                    <td><code>{contactfield=fax}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.firstname'|trans }}</td>
+                    <td><code>{contactfield=firstname}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.foursquare'|trans }}</td>
+                    <td><code>{contactfield=foursquare}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.googleplus'|trans }}</td>
+                    <td><code>{contactfield=googleplus}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.instagram'|trans }}</td>
+                    <td><code>{contactfield=instagram}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.ipaddress'|trans }}</td>
+                    <td><code>{contactfield=ipAddress}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.lastname'|trans }}</td>
+                    <td><code>{contactfield=lastname}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.linkedin'|trans }}</td>
+                    <td><code>{contactfield=linkedin}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.mobile'|trans }}</td>
+                    <td><code>{contactfield=mobile}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.phone'|trans }}</td>
+                    <td><code>{contactfield=phone}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.position'|trans }}</td>
+                    <td><code>{contactfield=position}</code></td>
+                </tr>
+                <tr>
+                    <td>{{ 'mautic.placeholder_tokens.contact.skype'|trans }}</td>
+                    <td><code>{contactfield=skype}</code></td>
+                </tr>
```

Comment:
These tokens really need to be dynamically generated. What if a user creates another custom field? They would be confused why it's not here. What if a plugin enables more tokens? How would it add it into this list?

Reply:
This topic is problematic because of these issues:
- you can't dynamically generate the entire content of this modal
- Mautic seems not to have a built-in way to automatically parse custom date fields following the system date format, so manually writing the token is needed for now (e.g. `{contactfield=aliasofyourcustomdatefield|time}`)
- AJAX calls only show tokens for builders
- AJAX calls with the builder tokens don't always show all tokens (like when you don't have an Asset created yet), leading to things appearing and disappearing, which is a real UX issue I'm fixing everywhere
- There are many more tokens than those available in email or page builders, please interact with the modal to see the full scope
- there are tokens available for specific fields across the platform (like the Redirect URL field for forms)
- Do you know that there's a  `'{trackable=(.*?)}'`, token? I found it watching the PR from another dev and I have no idea how it's used as can't find in docs; that's truly problematic)

> What if a plugin enables more tokens?

I hope someone can create a function that parse new `<li>` items for the section "Other tokens" as already happens with custom field groups on the interface, but that's out of scope for my PR

![image](https://github.com/user-attachments/assets/359951ca-751b-4198-b0a1-6dc40758c244)

I left an accordion template for Twig to make this process even easier than having to handle raw HTML

> What if a user creates another custom field?

I can expand the part that talks about custom fields, as this modal is help content

![image](https://github.com/user-attachments/assets/36f732c7-8fd8-40f3-be01-5f44b8302d66)